node/src/effect.rs

```diff
@@ -1362,16 +1362,13 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
-    /// Retrieves finalized deploys from blocks that were created more recently than the TTL.
-    pub(crate) async fn get_finalized_deploys(
-        self,
-        ttl: TimeDiff,
-    ) -> Vec<(DeployHash, DeployHeader)>
+    /// Retrieves finalized blocks that were created more recently than the deploy TTL.
```

Comment:
Since there's no `Deploy` argument to this method I don't think we should be mentioning it in the comment.

Reply:
Right, that should say "maximum deploy TTL". We should also explain that that means it will contain the hashes of all deploys that are finalized but not yet expired.